### ISSUE TITLE
Clicking on Dashboard or Reports does not expand the side menu on a tablet

### ISSUE THREAD

**Issue by marthakeezy** [msupply-foundation/open-msupply#4969]
## What went wrong? 😲

Clicking on any item in the side menu that has sub-menus expands the side menu, however `Dashboard` or `Reports` (which is a single menu item) does not produce the same behaviour. 

https://github.com/user-attachments/assets/e3745c13-0a43-4fd6-a561-45ff2eb60f89

## Expected behaviour 🤔

All items in the side menu should have the same behaviour - it's useful for the user to be able to see the menu items before clicking on any specific ones. There is no way of knowing the second item is `Reports` unless you click elsewhere for example

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Log into your store
2. Click on `Dashboard or Reports`
3. See the side menu does not expand
4. Click on `Distribution` -> see menu expands without going into the module itself

## Your environment 🌱

- Open mSupply Version: `V2.3.0-RC4`
- Legacy mSupply Central Server Version: `V7.17.10`

- Platform: Tablet (apk)

- Database type: SQLite3

**Comment by mark-prins** [comment]
By design - the other menu items expand because the top level (Distribution, Replenishment etc) don't navigate anywhere. the dashboard / reports _do_ navigate and so they behave differently

**Comment by marthakeezy** [comment]
It seems counter-intuitive to me when users can click on a button and it does not expand to let them know what they clicked on. If someone navigates to Reports, they can't actually see they are in that area :0 Is that intended?

**Comment by mariyamsupply** [comment]
Just clicked around.. I actually agree it feels weird. 
My suggestion would be wherever we click on the closed nav bar it just opens it up at the page where user is, so then they can click and go where they want.

**Comment by mark-prins** [comment]
> wherever we click on the closed nav bar it just opens it up at the page where user is

but if the nav bar is collapsed, and you click _dashboard_ or _reports_ then what happens? the nav bar remains open? I don't want the nav bar to pop open and keep me on the dashboard or reports page.

Also, if you hover over the closed nav bar then it opens.. or it should. I had it stay closed once, but now cannot recreate and it does this:

https://github.com/user-attachments/assets/dd39858d-624e-45d3-8682-fc921c69a930

Having the bar open on click instead of the hover is slightly more work / more annoying for users on desktop. And we still have the issue of what to do with the nav bar when you click the two top level menu items (dashboard & reports)

**Comment by marthakeezy** [comment]
Thanks Mark! Just wondering if the navigation bar can behave differently for different devices? Because if we are testing `tablet first` to imitate our users' experience, then you cannot hover over :(

**Comment by mark-prins** [comment]
yes, could do. it would still mean that you need to tap 5 of the menu items twice before anything would happen. You'd tap once, the menu would expand but it wouldn't navigate like it does now. Then you'd need to tap again to navigate. Willing to have a try, though I would find that annoying.

**Comment by mariyamsupply** [comment]
A prototype for Tablet to represent the suggested interaction with Nav:
https://www.figma.com/proto/QAItIAHPsOO8GwzaCzVixv/Testing?page-id=402%3A69630&node-id=402-70544&node-type=frame&viewport=441%2C815%2C0.08&t=6ZGGM3hKm38Byiy2-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=402%3A70544

**Comment by marthakeezy** [comment]
Caught up with Mariya in person, gave feedback on order of sub-menus and icons, otherwise looks fab! I do strongly believe we need to find a way to conduct user focus groups and research for someone to give us feedback outside our own teams/org
